Move External Identities APIs to v1.0 (#12210)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/resources/userflowlanguageconfiguration.md

---
title: "userFlowLanguageConfiguration resource type"

> description: "The userFlowsLanguageConfiguration object allows a user flow to support customization of multiple languages."

Reviewer: ```suggestion
description: "Allows a user flow to support multiple languages."
```

author: "jkdouglas"
localization_priority: Normal
ms.prod: "identity-and-sign-in"
doc_type: resourcePageType
---

# userFlowLanguageConfiguration resource type

Namespace: microsoft.graph

> User flows language customization is a feature that allows a given user flow to support customization of multiple languages.

Reviewer: ```suggestion
Allows a user flow to support the use of multiple languages.
```
